systemd/systemd.go

```diff
@@ -266,6 +266,10 @@ type Systemd interface {
 	// CurrentMemoryUsage returns the current memory usage for the specified
 	// unit.
 	CurrentMemoryUsage(unit string) (quantity.Size, error)
+	// CurrentTasksCount returns the number of tasks (processes, threads, kernel
+	// threads if enabled, etc) part of the unit, which can be a service or a
+	// slice.
+	CurrentTasksCount(unit string) (int, error)
```

Comment:
Negative task count does not make much sense IMO
```suggestion
	CurrentTasksCount(unit string) (uint, error)
```